docs/installation.md

```diff
@@ -25,6 +25,16 @@ The Lychee gallery has a few system requirements. You will need to make sure you
    		- Tokenizer
    		- XML
    		- ZIP
+   	- These PHP extensions are necessary if you are running a FreeBSD system:
+   	 	- Simplexml
+		- Dom
+   	 	- Session
+   	 	- Zlib
+   	- For database, you will need these PHP extensions:
+   	 	- SQLite3 (For Lychee's inbuilt SQLite3)
+   	 	- MySQLi (For MySQL or MariaDB)
+   	 	- PDO_MySQL (For MySQL or MariaDB)
```

Comment:
```suggestion
   	 	- MySQLi (For MySQL or MariaDB, recommended modern extension)
   	 	- PDO_MySQL (For MySQL or MariaDB, legacy fall-back extension in case MySQLi is not available)
```

The list sounds as if both extensions were required for MySQL simultaneously, but it is an either-or with bias towards MySQLi. I woner if there is any PHP version and or distribution which does not provide MySQLi, especially since our minimal supported version is PHP 8.

But I don't use FreeBSD myself, so I have no idea.

Reply:
During my installation, I install MySQLi only, and the installation check webpage is showing PDO_MySQL aslo required.